types/node/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/node",
-    "version": "20.8.9999",
+    "version": "20.9.9999",
```

Comment:
Why not `20.9.0`?
And what the sense to update version without any changes?

Reply:
9999 is required on DT; it's the way that packages in this repo are semantically newer than any on DT.

Reply:
See also:
- https://jakebailey.dev/posts/pnpm-dt-3/
- https://github.com/DefinitelyTyped/DefinitelyTyped#:~:text=because%20the%20version%20is%20listed%20as%2020.8.99999

Reply:
But yes, I don't understand why this PR bumps the version without actually including any new features, unless there really weren't any new features.

Reply:
The [CHANGELOG](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V20.md#20.9.0) suggest the only meaningful change is the transition to LTS.

Reply:
https://github.com/nodejs/node/blob/v20.x/src/undici_version.h
As I see `undici` only now was updated to 5.26. Current version in `package.json` is mistake (was copied from `main` branch - v21.x).

Reply:
Hm, Andrew's PR that did that bump showed that it matched at runtime...

Reply:
Yeah sorry, this commit refers to v20.8.1 release

Reply:
Can you mark this conversation as resolved if it has been satisfactorily addressed?

Reply:
Is there anything outstanding that would prevent an approval?

Reply:
Not from me; I was just answering the question as to the versioning. As far as I'm concerned, this is fine; nothing was added in 20.9 so the bump is harmless (though, in a way, useless).

Reply:
@jakebailey I have a script to enforce matching versions of `.nvmrc` and @types/node, which according to DefinitelyTyped are only allowed to differ in patch. I do not think of following a contract as useless 😉

Reply:
That sounds needlessly strict; you're basically just letting DT set the pace of getting bugfixes from Node. Upgrading cannot possibly cause harm; you just won't see features until they're added here.

Reply:
Bugfixes should be released as patches, and as such would not be blocked on `@types/node`.

While we don’t currently enforce it in any of our repo, I believe that highlighting inconsitencies between Node versions and type definitions has the following benefits:
1. it encourages users of `@types/nodes` to contribute to DefinitelyTyped to make sure they can get new minor version features
2. it discourages users from relying on `// @ts-expect-error` comments for missing features
3. it makes the relationship between TypeScript, Node and community definitions less obscure and more approachable